docs/INTERNAL_METRICS.md

```diff
@@ -0,0 +1,52 @@
+# LogDNA Agent Internal Metrics
```

Comment:
I've used the word "internal" to avoid misleading the user into thinking that the logdna agent gathers metrics from other pods.

Reply:
Maybe we could just have the "exposing Agent Metrics as prometheus endpoint" as the top level title? I'm not sure most customers realise the agent ships metrics as log lines

Reply:
I wanted to include the log lines metrics information here because it changes behaviour from the previous version. It can be unexpected change for users that are parsing that information (hopefully few users), but at least we have a doc page where we can redirect them.